docs/resources/certificate.md

```diff
@@ -27,7 +27,8 @@ resource "artifactory_remote_maven_repository" "my-remote" {
 The following arguments are supported:
 
 * `alias` - (Required) Name of certificate.
-* `content` - (Required) PEM-encoded client certificate and private key.
+* `content` - (Optional) PEM-encoded client certificate and private key.
+* `file` - (Optional) Path to the PEM file.
```

Comment:
These are mutually exclusive, right? I mean setting the content to `file(my.pem)` isn't exactly challenging

Reply:
Yep, I'll update the doc.